### DISCUSSION TITLE
Status 5 bei Nachforderung RTW

### DISCUSSION THREAD

**Discussion by derl30n** [LSS-Manager/LSSM-V.4#1637] [Q&A] [answered]
Moin ich hab da mal eine Frage.

In LSS v3 gab es ein Modul, welches dafür sorgte, dass bei Rettungsdienst Einsätzen ein entsprechendes Rettungsmittel einen Sprechwunsch äußerte, wenn benötigte Einsatzmittel nicht vor Ort waren.

(Ähnlich wie standartmäßig bei der Einsatzausbreitung automatisch eine Rückmeldung gegeben wird.)

Gibt es pläne dieses Feature wieder in LSS v4 anzubieten?

Es ist etwas lästig Einsätze zu micro-managen wenn z.b. eine Tragehilfe oder ein NEF "nachgefordert" wird.

Ich verstehe, dass es beim Vorgänger auch der Fall war, dass z.B. der RTW ein NEF nachgefordert hat obwohl es ebenfalls mit alamiert wurde. Mir persönlich ist das tatsächlich gleichgültig ob das jetzt auch ingame sein muss, in der Realität kommt das aber tag-täglich vor ;)

VG Leon

**Comment by jxn-30** [comment, marked as answer]
Ja, es ist geplant, alle Module aus der V3 auch in die V4 zu übernehmen – das braucht aber leider Zeit, bis wir das geschafft haben. Den aktuellen "Übernahmestand" findest du in unserem Wiki unter https://proxy.lss-manager.de/v4/docs/de_DE/apps.html#lssm-v-3-vs-lssm-v-4, aktuell werden aber leider nur Module gelistet, die in allen Sprachen in der V3 aktiviert sind.
Natürlich kannst du die V3 und V4 parallel nutzen, bis alle genutzten Module in die V4 übernommen sind.

LG
Jan

**Comment by Cr4zyc4k3** [comment]
Nur noch als Hinweis. Es handelt sich um den LeitStellenSpielManager, dementaprechend sind die Abkürzungen LSS**M** Version 3 & 4
